lib/membrane_live/room.ex

```diff
@@ -303,44 +306,24 @@ defmodule MembraneLive.Event do
     {:noreply, state}
   end
 
-  def handle_info({:timer_action, action}, %{timer: timer} = state) do
-    case Timer.handle_action(timer, action,
-           timeout: MembraneLive.get_env!(:last_peer_timeout_long_ms)
-         ) do
-      {:ok, timer} ->
-        {:noreply, %{state | timer: timer}}
+  @impl true
+  def handle_info(:close_room, state) do
+    result = Engine.terminate(state.rtc_engine, timeout: 10_000, force?: true)
```

Comment:
Could we move this number to the module attribute?

Reply:
there is the `@timeout` parameter but I forgot to change this.